src/index.html

```diff
@@ -11,7 +11,115 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="info">
```

Comment:
better naming form-group, form-block...

Reply:
<h1> deleted, it doesn't need

Reply:
The comment applies only to the last line or selected lines